### PR TITLE
Add namespace explicitly

### PR DESCRIPTION
**Pull request by CiraciNicolo**
The rationale behind this PR is that there are cases where it becomes quite complicated to deploy the application unless the namespace is explicitly provided — for example, when managing the application using a GitOps approach.

### CODE REVIEW

File: charts/cloudnative-pg/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: cloudnative-pg
 description: CloudNativePG Operator Helm Chart
 icon: https://raw.githubusercontent.com/cloudnative-pg/artwork/main/cloudnativepg-logo.svg
 type: application
-version: "0.22.1"
```
Comment (itay-grudev) [inline charts/cloudnative-pg/Chart.yaml:21]: Could you please remove this? It contradicts our release process.

File: charts/cluster/Chart.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ name: cluster
 description: Deploys and manages a CloudNativePG cluster and its associated resources.
 icon: https://raw.githubusercontent.com/cloudnative-pg/artwork/main/cloudnativepg-logo.svg
 type: application
-version: 0.1.0
```
Comment (itay-grudev) [inline charts/cluster/Chart.yaml:21]: Same thing here. Please remove it.

**Review by itay-grudev — CHANGES_REQUESTED** [review]

**Review by itay-grudev — COMMENTED** [review]
Could you also look at the test failures? Example:

```
Error: template: cluster/templates/pooler.yaml:7:16: executing "cluster/templates/pooler.yaml" at <include "cluster.namespace" .>: error calling include: template: cluster/templates/_helpers.tpl:5:16: executing "cluster.namespace" at <.Values.namespaceOverride>: nil pointer evaluating interface {}.namespaceOverride
```

**Comment by CiraciNicolo** [comment]
I don't understand the DCO failure, the commits are signed.

**Comment by itay-grudev** [comment]
One of your commits isn't, but you can manually set it to pass from [here](https://github.com/cloudnative-pg/charts/runs/34120488217).

**Comment by CiraciNicolo** [comment]
No, all my commits are signed. If you notice the error is quite strange:

```
Summary

Commit sha: [ce9fad6](https://github.com/cloudnative-pg/charts/pull/461/commits/ce9fad60ab1d38a26188117ca187bb07ac007485), Author: Nicolò Ciraci, Committer: Nicolò Ciraci; Expected "Nicolò Ciraci ciraci.nicolo@gmail.com", but got "Nicolò Ciraci [ciraci.nicolo@gmail.com](mailto:ciraci.nicolo@gmail.com)".
```

Screenshot 2024-12-10 at 09 26 41

Am I missing something obvious?

**Comment by itay-grudev** [comment]
No matter. This is good enough for me.

**Pull request merged**